crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -521,20 +521,26 @@ pub struct AuthorityEpochTables {
     /// Records messages processed from other nodes. Updated when receiving a new dkg::Message
     /// via consensus.
     pub(crate) dkg_processed_messages_v2: DBMap<PartyId, VersionedProcessedMessage>,
+    /// This table is no longer used (can be removed when DBMap supports removing tables)
+    #[allow(dead_code)]
     #[deprecated]
-    pub(crate) dkg_processed_messages: DBMap<PartyId, dkg_v0::ProcessedMessage<PkG, EncG>>,
+    pub(crate) dkg_processed_messages: DBMap<PartyId, Vec<u8>>,
```

Comment:
Is there a different preferred default value to use here?

Reply:
@mystenmark is it safe to change types like that or will this make it fail to open the db?